examples/seismic/self_adjoint/time_blocking.ipynb

```diff
@@ -0,0 +1,1365 @@
+{
```

Comment:
<p>reminder:</p><p> </p><blockquote> (link to nb?)</blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/devitocodes/devito/pull/1424/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
There is no checkpoint nb, right? I need guidance on what to link to I think.

Reply:
complete